tests/smoke/test_deeprec_model.py

```diff
@@ -21,13 +24,17 @@ def resource_path():
 @pytest.mark.gpu
 @pytest.mark.deeprec
 def test_model_xdeepfm(resource_path):
-    data_path = os.path.join(resource_path, '../resources/deeprec/xdeepfm')
-    yaml_file = os.path.join(data_path, r'xDeepFM.yaml')
-    data_file = os.path.join(data_path, r'sample_FFM_data.txt')
-    output_file = os.path.join(data_path, r'output.txt')
+    data_path = os.path.join(resource_path, "..", "resources", "deeprec", "xdeepfm")
+    yaml_file = os.path.join(data_path, "xDeepFM.yaml")
+    data_file = os.path.join(data_path, "sample_FFM_data.txt")
+    output_file = os.path.join(data_path, "output.txt")
 
     if not os.path.exists(yaml_file):
-        download_deeprec_resources(r'https://recodatasets.blob.core.windows.net/deeprec/', data_path, 'xdeepfmresources.zip')
+        download_deeprec_resources(
+            "https://recodatasets.blob.core.windows.net/deeprec/",
+            data_path,
+            "xdeepfmresources.zip",
+        )
```

Comment:
We don't have docstrings for this function. Also, the functions under *reco_utils/recommender/deeprec* either don't have docstring nor don't follow our format.

Reply:
Yeah you are right, there is a lot to refactor in deeprec. I'll start working on that. on this branch: `miguel/fixes_prerelease`

As for the tests, there is no need to add docstrings there. Some examples [1](https://github.com/Microsoft/LightGBM/blob/master/tests/python_package_test/test_consistency.py), [2](https://github.com/Microsoft/LightGBM/blob/master/tests/python_package_test/test_basic.py), [3](https://github.com/pytorch/pytorch/blob/master/test/test_torch.py)